src/sql/azdata.d.ts

```diff
@@ -3495,21 +3509,7 @@ declare module 'azdata' {
 	/**
 	 * Editor component for displaying the text code editor
 	 */
-	export interface EditorComponent extends Component {
-		/**
-		 * The content inside the text editor
-		 */
-		content: string;
-		/**
-		 * The languge mode for this text editor. The language mode is SQL by default.
-		 */
-		languageMode: string;
-		/**
-		 * The editor Uri which will be used as a reference for VSCode Language Service.
-		 * Currently this is auto-generated by the framework but can be queried after
-		 * view initialization is completed
-		 */
-		readonly editorUri: string;
+	export interface EditorComponent extends Component, EditorProperties {
```

Comment:
Were these fields intentionally removed? I didn't see any replacements

Reply:
These are defined in EditorProperties